Backpacks.cs

```diff
@@ -652,21 +674,84 @@ public void Open(BasePlayer looter)
                     return;
                 }
 
+                if (hasOverflowingItems && looter.userID == OwnerId)
+                {
+                    HandleOverflowingItems(looter);
+                }
+
                 PlayerLootContainer(looter, _itemContainer);
 
                 Interface.CallHook("OnBackpackOpened", looter, OwnerId, _itemContainer);
             }
 
-            public void ForceClose()
+            private void HandleOverflowingItems(BasePlayer receiver)
             {
-                foreach (BasePlayer looter in _looters.ToArray())
+                // Close for other looters since we are going to alter the capacity
```

Comment:
Maybe we can re-open the backpack for those looters after altering the capacity?

Reply:
I suppose we could, but that case would happen so rarely, and the impact is so negligible that I don't think it's worth the extra lines of code.

Reply:
Fair enough, I just thought it'd be confusing the the admin but yes, the chance of this happening is quite slim.